frontend/packages/ux-editor/src/components/rightMenu/DynamicsTab.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useContext, useEffect } from 'react';
+import { Button, ButtonColor, ButtonSize, ButtonVariant } from '@digdir/design-system-react';
+import { DynamicContent } from '../config/expressions/DynamicContent';
+import { PlusIcon } from '@navikt/aksel-icons';
+import { useText } from '../../hooks';
+import { FormContext } from '../../containers/FormContext';
+import { ExpressionElement } from "../config/expressions/ExpressionContent";
+import { ExpressionPropertyBase, ExpressionPropertyForGroup } from "../../types/Expressions";
+import { LayoutItemType } from "../../types/global";
+
+export interface Dynamic {
+  id?: string;
+  editMode: boolean;
+  property?: ExpressionPropertyBase | ExpressionPropertyForGroup; // action?
+  expressionElements?: ExpressionElement[];
+}
+
+export const DynamicsTab = () => {
+  const { form, formId } = useContext(FormContext);
+
+  if (!formId || !form) return null;
```

Comment:
This line should be moved below all the hook calls to avoid all the linting error messages. This also causes a lot of error messages in the console.